tests/test_star_fusion.py

```diff
@@ -134,3 +177,175 @@ def test_star_fusion_v060_01(star_fusion_v060_01, attrs, exp):
 def test_star_fusion_v060_02(star_fusion_v060_02, attrs, exp):
     assert getattr_nested(star_fusion_v060_02.json, attrs) == exp, \
         ", ".join([repr(x) for x in attrs])
+
+
+def test_star_fusion_v160_dummy_types(star_fusion_v160_dummy):
+    """ Test whether if the data fields have the correct type """
+    for result in star_fusion_v160_dummy.json:
+        # Test int
+        for field in ("nJunctionReads", "nSpanningFrags"):
+            assert isinstance(result[field], int)
+
+        # Test float
+        assert isinstance(result['FFPM'], float)
```

Comment:
Just a small thing: could the single quotes be replaced with double-quotes for consistency? This applies also for the next two lines.